draft-ietf-quic-transport.md

```diff
@@ -2227,6 +2241,9 @@ enable an attack on migration; see {{off-path-forward}}.
 An endpoint MUST expand datagrams that contain a PATH_RESPONSE frame to at
 least the smallest allowed maximum datagram size of 1200 bytes. This verifies
 that the path is able to carry datagrams of this size in both directions.
+However, an endpoint MUST NOT expand the PATH_RESPONSE if it is constrained
+by an anti-amplification limit.  This will only occur if the PATH_CHALLENGE
+is not send in an expanded packet.
```

Comment:
I like this, because this requirement is in symmetry with that being proposed for PATH_CHALLENGE.

I had [something different](https://mailarchive.ietf.org/arch/msg/quic/hUck4IjyiT0LNkKPy-gPsPeR_XA/) in my mind, but I prefer what this PR states.